packages/contracts/contracts/TellerV2Storage.sol

```diff
@@ -41,7 +42,7 @@ struct Payment {
 struct Bid {
     address borrower;
     address receiver;
-    address lender;
+    address _lender; // DEPRECATED
```

Comment:
However if we do end up storing the 'lender' inside of the lender manager storage, we should really consider structuring it more like a NFT. Now is the time to do it.  We should make it follow the ERC721 protocol so that loans can be transferred like an NFT optionally in the future...!!!